OpenSim/Simulation/SimbodyEngine/Joint.h

```diff
@@ -322,8 +328,8 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Joint, ModelComponent);
     /** Utility for derived Joints to add Coordinate(s) to reflect its DOFs.
     Derived Joints must construct as many Coordinates as reflected by the
     Mobilizer Qs. */
```

Comment:
The comment should be updated and document the additional argument. A constructed Coordinate requires the enum to identify its DOF in the Joint to ensure its list of Coordinates is consistent with its DOFs. Perhaps also mention that if their is a mismatch with the enumerator and the order in the Joint's list (when the Coordinate is added) an Exception is thrown.